apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx

```diff
@@ -42,19 +42,19 @@ export const ChooseSetupTypeScreen = () => {
       return
     }
 
-    navigation.navigate('setup-wallet-choose-network')
+    navigation.navigate('setup-wallet-create-choose-network')
   }
 
   const handleRestore = () => {
     setUpTypeChanged('restore')
 
     if (isProduction()) {
```

Comment:
can we have a comment here, "On production the step of network is skipped" the first thing I think when I read it, is that a dev test was letting behind ;)